### ISSUE TITLE
Sanity check supermarkets data

### ISSUE THREAD

**Issue by Robinlovelace** [cyipt/actdev#57]
See here: https://rpubs.com/RobinLovelace/721779

Think a quick look and a comment can lead this to be closed quickly.

**Comment by mvl22** [comment]
I assume that file is created by:
https://github.com/cyipt/actdev/blob/main/code/pois/getdata.R

If so, I don't actually understand how you're getting any results at all - the tag is wrong:

`createPoisDataset (area, "supermarkets.geojson", "shop", "supermarkets")`

The tag is `shop=supermarket` (not pluralised): https://wiki.openstreetmap.org/wiki/Tag:shop%3Dsupermarket

Good testcases would be to ensure that these three appear (1 Node example, 2 Way examples):

https://www.openstreetmap.org/node/1573848802
https://www.openstreetmap.org/way/126358284
https://www.openstreetmap.org/way/42867737

**Comment by Robinlovelace** [comment]
> I assume that file is created by:
> https://github.com/cyipt/actdev/blob/main/code/pois/getdata.R

As documented in the commit that closed the original issue https://github.com/cyipt/actdev/issues/15 the search term used in the query is:

https://github.com/cyipt/actdev/blob/c24dae480485aee397cafd24a3b0b6af38712cbf/code/get_pois.R#L5



> Good testcases would be to ensure that these three appear (1 Node example, 2 Way examples):
> 
> https://www.openstreetmap.org/node/1573848802
> https://www.openstreetmap.org/way/126358284
> https://www.openstreetmap.org/way/42867737

Are those supermarkets not in the map shared?

**Comment by mvl22** [comment]
OK, so it looks to me like Nodes are coming through, but not Ways (i.e. large ones are being omitted.)

... which would be explained by `SELECT * FROM 'points'`.

**Comment by Robinlovelace** [comment]
> https://www.openstreetmap.org/node/1573848802

Is this not it:

![image](https://user-images.githubusercontent.com/1825120/107048190-910f1a80-67c0-11eb-8e65-8215070e9f89.png)



> Which would be explained by "SELECT * FROM 'points'"...

Polygons (ways with the same start and end point) are in there.

**Comment by mvl22** [comment]
But the tag itself is on the Way, not the Points included on the Way.

I.e.

Small supermarkets like https://www.openstreetmap.org/node/1573848802 are a single node with tag shop=supermarket.

Large supermarkets like https://www.openstreetmap.org/way/126358284 are a single way with tag shop=supermarket, with that way containing nodes containing no/irrelevant tags.

As such it's likely that the current script is basically picking up local biggish food stores that are local foot traffic generators, but not large supermarkets which are significant trip generators from a wide area.

**Comment by Robinlovelace** [comment]
> https://www.openstreetmap.org/way/126358284

Can you identify false negatives? This seems like a 'positive positive':

![image](https://user-images.githubusercontent.com/1825120/107048643-17c3f780-67c1-11eb-8351-fdbc7c3ac353.png)

**Comment by mvl22** [comment]
> https://www.openstreetmap.org/way/126358284

is Asda (tagged using Ways), located at the bottom of the map example you gave, on the other side of Coldham's Lane to the point shown.

The point shown on the map is a medium-sized Lidl, which is one of two shops in a single (internally-partitioned) building, and as such the supermarket is a node.

So this matches my theory that your script isn't yet catching shop=supermarket marked on a Way.

**Comment by Robinlovelace** [comment]
Good point @mvl22. It helps if you're specific - you're right this is not the ASDA:

![image](https://user-images.githubusercontent.com/1825120/107049658-47273400-67c2-11eb-9ebd-b4f7520ecaad.png)

**Comment by mvl22** [comment]
> It helps if you're specific

It's worth noting that the three URLs I gave do specifically state way/node in them. I perhaps forget as a web developer that not everyone is used to looking at URL structure.
